packages/entrypoint/modules/entrypoint/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "matbas_build" {
+  description = "Build type - either debug or release"
+}
+
+variable "path" {
+  description = "The path to write the entry point"
```

Comment:
Great documentation! lol

I currently don't run `tflint`. I should probably do that!